example/app/data/DemoDataSource/recipes/plugins/view_selector/car_infosys/owner.recipe.json

```diff
@@ -41,8 +42,7 @@
                 "type": "PLUGINS:dm-core-plugins/form/FormInput",
                 "fields": ["earlierOwners"]
               }
-            },
-            "eds_icon": "group"
+            }
```

Comment:
Should leave one of these icons in the ViewConfig, to make sure the backwards compatibility works

Reply:
Fixed